bluemira/base/parameter_frame/_parameter.py

```diff
@@ -137,12 +137,28 @@ def value(self) -> ParameterValueType:
     def value(self, new_value: ParameterValueType):
         self.set_value(new_value, source="")
 
-    def value_as(self, unit: Union[str, pint.Unit]) -> ParameterValueType:
-        """Return the current value in a given unit"""
+    def value_as(self, unit: Union[str, pint.Unit]) -> Union[ParameterValueType, None]:
```

Comment:
😆 MyPy is not going to like this... Not sure that we have any other option though